integration-tests/subscriptions.test.ts

```diff
@@ -86,6 +86,90 @@ if (SUBSCRIPTIONS_ENABLED === 'true') {
                 expect(notifications).toEqual([]);
             });
         }
+
+        test('end to end test with id notifications', async () => {
+            const uuid = v4();
+            const subResource = randomSubscription(uuid);
+            // 1. Create a Subscription.
+            const postSubscriptionResult = await client.post('Subscription', subResource);
+            expect(postSubscriptionResult.status).toBe(201);
+            // 2. Create/Update a resource that matches the subscription.
+            const postPatientResult = await client.post('Patient', resourceThatMatchesSubscription);
+            expect(postPatientResult.status).toEqual(201);
+            // 3. Verify that notifications are received
+            // give 2 minutes for notification to be placed in ddb table
+            await new Promise((r) => setTimeout(r, 120_000));
+            // make sure notification was receieved
+            let notifications = await subscriptionsHelper.getNotifications(
+                `/${uuid}/Patient/${postPatientResult.data.id}`,
+            );
+            expect(notifications).not.toEqual([]);
+            expect(notifications[0].httpMethod).toEqual('PUT');
+            expect(notifications[0].body).toBeNull();
```

Comment:
This would be faster using `waitForExpect`. That way you don't need to wait an entire minute.

Also, can you assert that the headers match?